File: pinyougou-parent/pinyougou-portal-web/src/main/java/com/pinyougou/portal/controller/ContentController.java
package com.pinyougou.portal.controller;

import com.alibaba.dubbo.config.annotation.Reference;
import com.pinyougou.content.service.ContentService;
import com.pinyougou.pojo.TbContent;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.List;

/**
 * <h3>pinyougou</h3>
 * <p>广告管理</p>
 *
 * @author : 沈疴
 * @date : 2020-11-24 15:09
 **/
@RestController
@RequestMapping("/content")
public class ContentController {

    @Reference
    private ContentService contentService;

    @RequestMapping("/findByCategoryId")
    public List<TbContent> findByCategoryId(Long categoryId){
        return contentService.findByCategoryId(categoryId);
    }
}

File: pinyougou-parent/pinyougou-portal-web/src/main/webapp/js/controller/contentController.js
app.controller('contentController',function ($scope, contentService) {

    //所有广告
    $scope.contentList=[];

    /*根据分类ID查询出对应广告,存入广告列表数组对应的元素中*/
    $scope.findByCategoryId=function (categoryId) {
        contentService.findByCategoryId(categoryId).success(
            function (response) {
                $scope.contentList[categoryId]=response;
            }
        )
    };


});